packages/postgres/src/dialect.test.ts

```diff
@@ -0,0 +1,23 @@
+import { Sequelize } from '@sequelize/core';
+import type { PostgresConnectionOptions } from '@sequelize/postgres';
+import { PostgresDialect } from '@sequelize/postgres';
+import { expect } from 'chai';
+
+describe('PostgresDialect#parseConnectionUrl', () => {
+  const dialect = new Sequelize({ dialect: PostgresDialect }).dialect;
+
+  it('parses connection URL', () => {
+    const options: PostgresConnectionOptions = dialect.parseConnectionUrl(
+      'postgres://user:password@localhost:1234/dbname?client_encoding=utf8mb4',
```

Comment:
We should also have this test for `postgresql` as protocol

Reply:
Added